ext/libxml/tests/bug76777.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Bug #76777 (first parameter of libxml_set_external_entity_loader callback undefined)
+--SKIPIF--
+<?php if (!extension_loaded('libxml')) die('skip'); ?>
+--FILE--
+<?php
+ini_set('error_reporting',PHP_INT_MAX-1);
```

Comment:
Usually, we do not set `error_reporting` unless there's a particular reason to do so, which doesn't seem to be the case here (also `PHP_INT_MAX-1` is quite uncommon to my knowledge).